### DISCUSSION TITLE
runtimeConfig is not a runtime config anymore?

### DISCUSSION THREAD

**Discussion by vedmant** [nuxt/nuxt#16134] [Questions]
In Nuxt 2 runtimeConfig was updating on each run from env variables, but this stopped working in Nuxt 3. So the questions then what is the right way to actually haver runtime config that's updated at app restart?

Here is reproduction example: https://stackblitz.com/edit/github-o8ojpu?file=nuxt.config.js

run yarn build then RELEASE_STAGE=staging node .output/server/index.mjs
see in console that releaseStage is not staging

**Comment by vedmant** [comment]
Ok, it appears that then only way is to use env variables names starting with NUXT_* to override runtimeConfig.
This is ridiculous to not allow to use any ENV variable name and to force everyone to use  NUXT_* from my opinion.

**Reply by simonsax** [reply]
I have the same problem. In production on a PaaS platform (platform as a service) things like database_uri are provided by environment variables (which naming I cannot control, and they don't start with NUXT :-)). Therefore I'm not able to set them after the build phase.

Did you find any solution yet?
